external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/hpo_config.yaml

```diff
@@ -4,12 +4,12 @@ hp_space:
   learning_parameters.learning_rate:
     param_type: qloguniform
     range:
-      - 0.0003
-      - 0.1
+      - 0.0014
```

Comment:
Could you revert learning rate as default value of template.yaml? We discarded bigger learning rate training schedule by performance degradation for C-IL experiments.